src/rtc_peer.ts

```diff
@@ -93,11 +113,7 @@ export class RTCPeer extends EventEmitter {
     }
 
     private onTrack(ev: RTCTrackEvent) {
-        if (ev.streams.length === 0) {
-            this.emit('stream', new this.webrtc.MediaStream([ev.track]));
-            return;
-        }
-        this.emit('stream', ev.streams[0]);
+        this.emit('stream', new MediaStream([ev.track]));
```

Comment:
This needs to be:
```suggestion
        this.emit('stream', new this.webrtc.MediaStream([ev.track]));
```

Reply:
Thanks, I forgot we are multi platform now :)